docs/install_guides/configurations-for-nvidia-gpu.md

```diff
@@ -0,0 +1,308 @@
+# Configurations for NVIDIA GPUs with OpenVINO™ {#openvino_docs_install_guides_configurations_for_nvidia_gpu}
+
+@sphinxdirective
+
+.. meta::
+   :description: Learn how to provide additional configuration for NVIDIA 
+                 GPUs to work with Intel® Distribution of OpenVINO™ toolkit 
+                 on your system.
+
+
+.. _nvidia gpu guide:
+
+OpenVINO™ NVIDIA GPU plugin enables deep neural networks inference on NVIDIA GPUs. 
+The plugin uses custom kernels and cuBLAS, cuDNN, cuTENSOR libraries as a backend 
+and requires prior configuration on a supported system.
+
+
+Linux
+####################
+
+OpenVINO™ NVIDIA GPU plugin is supported and validated on the following platform:
+
++------------------------+------------------------+----------------+
+| Operating System       | GPU                    | Driver version |
++========================+========================+================+
+| Ubuntu* 20.04 (64-bit) | NVIDIA Quadro RTX 4000 | 520.61.05      |
++------------------------+------------------------+----------------+
+
+
+OpenVINO™ NVIDIA GPU plugin is not included in OpenVINO™ toolkit. In order to use 
+the plugin you need to download and `install dependencies <install-dependencies>`__ and then 
+`build it from source code <build-the-plugin-with-cmake>`__. Follow the instructions below.
```

Comment:
I suppose it will be changed since 2023.1 release.
@evgenytalanin-intel @p-durandin ?

Reply:
I think it's full interpretation of the base article from openvino_contrib
https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/nvidia_plugin

And we need to remove build-related sections

Reply:
All the necessary changes have been implemented with the https://github.com/openvinotoolkit/openvino/pull/19016/commits/89468d400089ec825a60d511f3042837d189b4c8 commit.